src/scripts/components/foundations/Button.js

```diff
@@ -1,88 +1,43 @@
 import styled, { css } from 'styled-components'
+import {
+  BUTTON_HEIGHT,
+  BUTTON_BORDER_RADIUS,
+  BUTTON_PADDING_LEFT
+} from 'constants/UIConstants'
 
-export const ButtonStyleBackground = css`
+export const ButtonStyleText = css`
   ${props => {
-    let _background: String
+    let _size: String
 
-    if (props.white) {
-      _background = props.theme.colors.button.white
-    } else if (props.gray) {
-      _background = props.theme.colors.button.gray
-    } else if (props.pink) {
-      _background = props.theme.colors.button.pink
-    } else if (props.green) {
-      _background = props.theme.colors.button.green
-    } else {
-      _background =
-        'linear-gradient(135deg, ' +
-        props.theme.colors.VideoForm.info.progress.barFrom +
-        ' 0%, ' +
-        props.theme.colors.VideoForm.info.progress.barTo +
-        ' 100%)'
+    if (props.anchor) {
+      _size = null
     }
 
-    return css`
-      background: ${_background};
-`
-  }};
-`
-
-export const ButtonStyleColor = css`
-  ${props => {
-    let _color: String
-
-    if (props.white) {
-      _color = props.theme.colors.button.white
-    } else if (props.purple) {
-      _color = props.theme.colors.button.purple
-    } else if (props.pink) {
-      _color = props.theme.colors.button.pink
-    } else if (props.green) {
-      _color = props.theme.colors.button.green
+    if (props.big) {
+      _size = props.theme.fonts.button.big
+    } else if (props.small) {
+      _size = props.theme.fonts.button.small
     } else {
-      _color = props.theme.colors.button.gray
+      _size = props.theme.fonts.button.main
```

Comment:
instead of `.fonts` can this be `.fontSizes`? That seems to be more accurate

Reply:
It holds information about the family and weight of the typography too.

Reply:
Maybe we can change that and have separate keys for all of those properties?

Reply:
I've created an issue for that:
https://github.com/Paratii-Video/paratii-portal/issues/776